src/Tasks/Task-3 (Redux)/reducers/StateChange.js

```diff
@@ -0,0 +1,20 @@
+const initialState =0;
+
+const StateChange = (state=initialState , action) => {
+    switch (action.type){
+        case "INCREMENT":
```

Comment:
Manage variables for the attribute like INCREMENT and DECREMENT in another file and import here and use.